third_party/nvidia/hopper/include/Transforms/Passes.td

```diff
@@ -102,4 +102,12 @@ def NVGPUTestPingPongSync : Pass<"nvgpu-test-ping-pong-sync", "mlir::ModuleOp">
   ];
 }
 
+def NVGPUTest1DTMEMAlloc : Pass<"nvgpu-test-1D-tmem-alloc", "mlir::ModuleOp"> {
```

Comment:
I found it simpler to test by putting it in its own pass similar to PingPong. Fundamentally we don't need to put this in its own pass although it should be completely feasible.